README.md

```diff
@@ -42,49 +153,20 @@ yarn add @maidsafe/safe-node-app
 
 3. Install the dependencies:
 
-    ```bash
+    ``` bash
     cd safe_app_nodejs
     yarn
     ```
-
-### Testing
+#### Testing
 
 To run the tests locally, run `yarn test`.
 
 Note: If you are compiling your own [`safe_app`](https://github.com/maidsafe/safe_client_libs/tree/master/safe_app) library for testing purposes, and if you want to be able to run the tests, make sure to include `testing` in your build features when compiling `safe_app` in `safe_client_libs`, i.e. `cargo build --release --features "use-mock-routing testing"`.
 
-### Mobile Development
+#### Mobile Development
```

Comment:
For mobile development we will need the native libraries for android/iOS. I tried setting this flag and it skipped system_uri but it did not download the mobile native libraries.

Even if it **did** download the native libs, will I be able to use safe-node-app for mobile development? Has anyone tried this before?

Reply:
I don't think anyone has tried this, this was just a suggestion we received some time ago since we knew that systen_uri lib wouldn't be needed for mobile. I guess we need to work on this to make the native libs for mobile to be downloaded automatically. If you copy the native safe_app lib manually you should be able to use safe-app-nodejs for mobile.